demo/pages/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Svg from '@moxy/react-svg';
+
+import styles from './index.module.css';
+
+const twitterLogo = import(/* webpackChunkName: "svg-sprite" */ '../media/twitter.inline.svg');
+
+const Home = () => (
+    <div>
```

Comment:
Add a `className` here so you can have more specificity to be able to override the `<Svg />` className, check https://github.com/moxystudio/react-lib-template/pull/5 for reference.


```js
const Home = () => (
    <div className={ styles.home }>
        <h1>react-svg</h1>
        <br />
        <p>This is a svg of the Twitter logo.</p>
        <Svg svg={ twitterLogo } className={ styles.svg } />
    </div>
);
```